kernel/api/src/main/java/org/sakaiproject/tasks/api/Task.java

```diff
@@ -61,7 +61,7 @@
     @Column(name = "DESCRIPTION", length = 255, nullable = false)
     private String description;
 
-    @Column(name = "SYSTEM", nullable = false)
+    @Column(name = "\"SYSTEM\"", nullable = false)
```

Comment:
Does this work okay with Oracle?

Reply:
I think so but I don't have an instance to test right now, however, I've checked several references and all of them state that it works with Oracle, MySQL and PostgreSQL.

Worth trying in Oracle Nightly after the refresh?

Reply:
I think this may work too but I haven't tested it:

"[SYSTEM]"

Reply:
not a fan of this fix :(

why not just rename it?

Reply:
Not sure since this is in 21.x ? 

We have other options, use brackets like "[SYSTEM]" and there are also references to use this property:

hibernate.globally_quoted_identifiers=true

Reply:
> not a fan of this fix :(
> 
> why not just rename it?

See https://github.com/sakaiproject/sakai/pull/9313